.github/workflows/gradle-build-pr.yml

```diff
@@ -26,155 +26,72 @@ jobs:
-                kubectl create secret generic ${i}-inuse --from-literal=inuse=${i} -n ${i}
-                tail=$(echo $i | awk -F- {'print $2'})
-                echo "::set-env name=IT_JADE_API_URL::https://jade-${tail}.datarepo-integration.broadinstitute.org"
-                echo "::set-env name=NAMESPACEINUSE::${i}"
-                return 0
-              fi
-            done
-            sleep 60
-            find_namespace
-          }
-          find_namespace
-      # Build the Docker image
-      - name: Build new api container and push to gcr
-        env:
-          DEV_PROJECT: broad-jade-dev
-        run: |
-          GCR_TAG=$(git rev-parse --short HEAD)
-          echo "::set-env name=GCR_TAG::${GCR_TAG}"
-          ./gradlew jib
-      - name: Deploying new image to kubernetes
-        env:
-          DEV_PROJECT: broad-jade-dev
-          SECRET_CHART_VERSION: 0.0.4
-          DEPLOY_CHART_VERSION: 0.0.7
-        run: |
-          helm namespace upgrade ${NAMESPACEINUSE}-secrets datarepo-helm/create-secret-manager-secret --version=${SECRET_CHART_VERSION} \
-            --install --namespace ${NAMESPACEINUSE} -f \
-            "https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/integration/${NAMESPACEINUSE}/${NAMESPACEINUSE}Secrets.yaml"
-          helm namespace upgrade ${NAMESPACEINUSE}-jade datarepo-helm/datarepo --version=${DEPLOY_CHART_VERSION} --install \
-            --namespace ${NAMESPACEINUSE} -f \
-            "https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/integration/${NAMESPACEINUSE}/${NAMESPACEINUSE}Deployment.yaml" \
-            --set "datarepo-api.image.tag=${GCR_TAG}"
-      - name: "Build and test code"
-        env:
-          PGHOST: localhost
-          PGPORT: ${{ job.services.postgres.ports[5432] }}
-          PGPASSWORD: postgres
-          GOOGLE_CLOUD_PROJECT: broad-jade-integration
-          GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
-        run: |
-          # required for tests
-          psql -U postgres -f ./db/create-data-repo-db
-
-          ./gradlew assemble
-          ./gradlew check --scan
-          ./gradlew testConnected --scan
-          ./gradlew testIntegration --scan
-      - name: "Post test cleanup namespace lock"
+          actions_subcommand: 'gradlebuild'
+          role_id: ${{ secrets.ROLE_ID }}
+          secret_id: ${{ secrets.SECRET_ID }}
+      - name: "Deploy to cluster with Helm"
+        uses: broadinstitute/datarepo-actions@0.1.0
+        with:
+          actions_subcommand: 'helmdeploy'
+          helm_secret_chart_version: '0.0.4'
+          helm_datarepo_chart_version: '0.0.8'
+      - name: "Run Integration test via Gradle"
+        uses: broadinstitute/datarepo-actions@0.1.0
+        with:
+          actions_subcommand: 'gradleinttest'
+          pgport: ${{ job.services.postgres.ports[5432] }}
```

Comment:
Why is this here? The integration tests shouldn't be touching Postgres

Reply:
Would this include ditching the postgres service for the job?

Reply:
fixed

Reply:
removing that broke the test